tests/MongoDB.Driver.Tests/AuthenticationTests.cs

```diff
@@ -24,6 +24,8 @@
 using MongoDB.Bson;
 using MongoDB.Bson.Serialization.Serializers;
 using MongoDB.Bson.TestHelpers.XunitExtensions;
+using MongoDB.Driver.Core.Clusters.ServerSelectors;
+using MongoDB.Driver.Core.Connections;
```

Comment:
not used ns. Also, please remove all other not used `ns`s here

Reply:
Done